soccer/src/soccer/tutorial/SoccerMom.hpp

```diff
@@ -0,0 +1,22 @@
+#include <rj_constants/topic_names.hpp>
+#include <rj_msgs/msg/team_color.hpp>
+#include <rj_msgs/srv/quick_commands.hpp>
+#include <rj_msgs/srv/quick_restart.hpp>
+
+#include "rclcpp/rclcpp.hpp"
+#include "referee/referee_base.hpp"
+#include "std_msgs/msg/string.hpp"
+
+namespace tutorial {
+class SoccerMom : public rclcpp::Node {
+public:
+    SoccerMom();
+
+private:
+    std_msgs::msg::String tf;
```

Comment:
it's a little non-standard to keep messages as member variables; since they are not needed outside of the scope of the function that calls `publish`, there's no need to keep them here